New resource: `azurerm_eventgrid_partner_namespace` (#30266)
hashicorp/terraform-provider-azurerm: website/docs/d/eventgrid_partner_namespace.html.markdown

---
subcategory: "Messaging"
layout: "azurerm"
page_title: "Azure Resource Manager: azurerm_eventgrid_partner_namespace"
description: |-

>   Gets information about an existing EventGrid Partner Namespace

Reviewer: Could you make this change throughout the documentation? (This matches Microsoft's documentation)

I do realize there are other documentation pages that use `EventGrid` though in time these should be updated as well

```suggestion
  Gets information about an existing Event Grid Partner Namespace
```

---

# Data Source: azurerm_eventgrid_partner_namespace

Use this data source to access information about an existing EventGrid Partner Namespace

## Example Usage

```hcl
data "azurerm_eventgrid_partner_namespace" "example" {
  name                = "my-eventgrid-partner-namespace"
  resource_group_name = "example-resources"
}

output "eventgrid_partner_namespace_endpoint" {
  value = data.azurerm_eventgrid_partner_namespace.example.endpoint
}
```

## Arguments Reference

The following arguments are supported:

* `name` - (Required) The name of the EventGrid Partner Namespace resource.

* `resource_group_name` - (Required) The name of the resource group in which the EventGrid Partner Namespace exists.

## Attributes Reference

The following attributes are exported:

* `id` - The ID of the EventGrid Partner Namespace.

* `location` - The Azure Region where the EventGrid Partner Namespace exists.

* `endpoint` - The endpoint for the EventGrid Partner Namespace.

> * `partner_registration_id` - The fully qualified ARM Id of the partner registration that should be associated with this partner namespace.

Reviewer: Generally we just use `resource Id` and since this is an attribute can we use `associated with` rather than `that should be ....`

```suggestion
* `partner_registration_id` - The resource Id of the partner registration associated with this Event Grid Partner Namespace.
```
